Lib/test/test_typing.py

```diff
@@ -1839,6 +1857,17 @@ def test_eq_hash(self):
         self.assertNotEqual(C, Callable[..., int])
         self.assertNotEqual(C, Callable)
 
+    def test_dir(self):
+        Callable = self.Callable
+        dir_items = set(dir(Callable[..., int]))
+        for required_item in [
+            '__args__', '__parameters__', '__origin__',
+            # TODO: add to `collections.abc.Callable`:
+            # 'copy_with',
```

Comment:
I disagree with this TODO: see my analysis on the issue: https://github.com/python/cpython/issues/103395#issuecomment-1501160317